app/assets/javascripts/modules/repositories/components/tags/tags-table-row.vue

```diff
@@ -2,13 +2,22 @@
   .selected td {
     background-color: #e8f1f6;
   }
+
+  .vulns-link {
+    text-decoration: none;
+  }
+
+  .image-id {
```

Comment:
Instead of this, maybe we could use the `.text-monospace` class.

Reply:
Indeed.